docs/trust-list.mdx

```diff
@@ -26,13 +26,44 @@ The [contentcredentials.org](https://contentcredentials.org/) site hosts the fol
 - **The temporary known anchor list** in https://contentcredentials.org/trust/anchors.pem contains the list of known anchor certificates. If an end-entity [certificate's chain](getting-started.mdx#signing-and-certificates) can be traced back to an anchor certificate on this list, the certificate is considered "known."
 - **The configuration file**, https://contentcredentials.org/trust/store.cfg, specifies the [Extended Key Usage (EKU)](https://datatracker.ietf.org/doc/html/rfc9336) values accepted for end-entity certificates. An end-entity certificate must have at least one of the EKUs in this list to be valid.
 
+## Checking your certificate
+
+Before requesting to add your signing certificate to the known certificate list, perform a preliminary "sanity check" on your certificate by following these steps:
+
+1. **Ensure that signing with the certificate doesn't have any validation errors** by using a C2PA Tool command like this:
```

Comment:
This numbered list is rendering as `1.` for each entry. Numbering them all as `1.` normally works, but I believe you have to indent the body (so the code block and the paragraph that follows) in order for it to be considered part of the list item.

Reply:
Oops, fixed

Reply:
I don't immediately see the changes reflected in the PR, they still look unindented to me, but I'll wait for the deploy preview to be generated

Reply:
Yeah looks like it's still rendering as ones for every list item in the deployment preview.

![Screenshot 2025-02-05 at 2 26 04 PM](https://github.com/user-attachments/assets/85d1f997-0f73-442b-be6f-3a69952a8d5f)

Reply:
It's weird, because I made the necessary changes and tested them locally. I know Kozma set up a linter that sometimes does weird things to .md files in this repo, maybe that was the culprit.  

I made the same changes again, and it appears to be fixed now, looking at https://deploy-preview-188--cai-open-source.netlify.app/docs/verify-known-cert-list/#procedure.

But this makes me nervous....

Reply:
Very odd, definitely fixed now though!